components/org.wso2.carbon.identity.recovery/src/main/java/org/wso2/carbon/identity/recovery/util/Utils.java

```diff
@@ -1124,4 +1125,42 @@ public static boolean isUseVerifyClaimEnabled() {
         return Boolean.parseBoolean(IdentityUtil.getProperty
                 (IdentityRecoveryConstants.ConnectorConfig.USE_VERIFY_CLAIM_ON_UPDATE));
     }
+
+    /**
+     * Trigger recovery event.
+     *
+     * @param map       Property map.
```

Comment:
Shall we make the variable map more meaningful? Eg: properties 
Also, shall we order the parameters to start with the String?